GliaWidgets/CallVisualizer/TemporaryCommandFIle.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
```

Comment:
This file is also used by the secure conversations project, correct? Is that why it's called TemporaryCommandFile? At some point they both have to be merged into a single one. If that is the case, then it would be good to have a small ticket to remember that right? As the secure conversations project will be merged to master later than call visualizer, then I should create it in our project. Let me know if that's the case so that I can create it.